### DISCUSSION TITLE
How to get the standard output when calling cli.ImageBuild

### DISCUSSION THREAD

**Discussion by zhangzhenyuyu** [moby/moby#43792] [Q&A]
When I call `docker build -f * -t * .` ， I will get the following output

<img width="675" alt="image" src="https://user-images.githubusercontent.com/54766833/178139910-a6a2e475-840d-4fe7-82b4-87cb3e7af5c3.png">

When I call ImageBuild, how should I get output like above？thanks！

**Comment by nibaozhu8** [comment]
try this:
```
sudo DOCKER_BUILDKIT=1 docker build -f Dockerfile -t v1.x.x .
```
